bokehjs/src/lib/models/ui/dialog.ts

```diff
@@ -19,6 +19,9 @@ import {Or, Ref} from "core/kinds"
 import dialogs_css, * as dialogs from "styles/dialogs.css"
 import icons_css from "styles/icons.css"
 
+// Make sure this at least an order of magnitude lower than --bokeh-top-level.
+const base_z_index = 1000
```

Comment:
Could we have an assert here to verify this is lower than bokeh-top-level?

Reply:
It could be done here, by creating an element with base stylesheet and using `getComputedStyle()`, but what I intend to do (in another PR), is to make this a CSS variable and make sure all magic constants are in one place.